IsogeomGenerator/vol.py

```diff
@@ -734,16 +636,20 @@ def _compare_surfs(self, v1, v2):
                         self.surf_curve[s2].append(curve)
                         self.surf_curve[surf].append(curve)
 
-                        # remove merged verts and tris from each already existing surf
+                        # remove merged verts and tris from each already
+                        # existing surf
                         for vert_delete in s2_match_eh:
 
```

Comment:
why a blank line there ?

Reply:
This has to be refactored. I am not going to make edits to this code beyond what is necessary for getting a working user script.